src/components/browse/components/SelectedItemsController.js

```diff
@@ -129,42 +104,38 @@ export class SelectedItemsController extends React.PureComponent {
         // Kept for reference in case we want to re-introduce constrain that for 'select' button(s) to be visible in search result rows, there must be parent window.
         //var isThereParentWindow = inSelectionMode && typeof window !== 'undefined' && window.opener && window.opener.fourfront && window.opener !== window;
 
-        if (inSelectionMode) {
```

Comment:
This arrangement of `if` conditions is just a bit redundant, since we already return out if `!inSelectionMode`